library/ajax/drug_autocomplete/search.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Rather than embed assets (jquery and jquery-ui), instead use the ones already in public/assets